README.md

```diff
@@ -34,34 +36,46 @@ This means that Linux users receive pre-release builds earlier but Upscayl itsel
   
 </div>
 
-
 # 👨‍💻 Installation 
 
 ### 🐧 Linux 
 
-<a href="https://flathub.org/apps/org.upscayl.Upscayl">
-<img src="https://dl.flathub.org/assets/badges/flathub-badge-en.svg" height="50px"/>
-</a>
+  <a href="https://flathub.org/apps/org.upscayl.Upscayl">
+    <img src="https://dl.flathub.org/assets/badges/flathub-badge-en.svg" height="50px"/>
+  </a>
 
-<a href="https://appimage.github.io/Upscayl/">
-<img src="https://user-images.githubusercontent.com/25067102/191270389-9de37c0f-39a8-41f1-a659-8dd4e7b8ac28.png" height="50px"/>
-</a>
+  <a href="https://appimage.github.io/Upscayl/">
+    <img src="https://user-images.githubusercontent.com/25067102/191270389-9de37c0f-39a8-41f1-a659-8dd4e7b8ac28.png" height="50px"/>
+  </a>
 
-<a href="https://aur.archlinux.org/packages/upscayl-bin">
-<img src="https://user-images.githubusercontent.com/25067102/191269445-87050a77-c304-4284-9ea0-699721309c59.png" height="50px"/>
-</a>
+  <a href="https://aur.archlinux.org/packages/upscayl-bin">
+    <img src="https://user-images.githubusercontent.com/25067102/191269445-87050a77-c304-4284-9ea0-699721309c59.png" height="50px"/>
+  </a>
 
-<a href="https://snapcraft.io/upscayl/">
-<img src="https://snapcraft.io/static/images/badges/en/snap-store-black.svg" height="50px"/>
-</a>
+  <a href="https://snapcraft.io/upscayl/">
+    <img src="https://snapcraft.io/static/images/badges/en/snap-store-black.svg" height="50px"/>
+  </a>
+
+#### Flatpak
+
+1. Go to your distribtuion's software center (or your favourite installed one) and search for Upscayl.  
+Alternatively you can go to [Flathub/Upscayl](https://flathub.org/apps/org.upscayl.Upscayl) and click "Install".
+
+2. Download Upscayl using your distribution's software center
+
+*Please note:*
+- *On some distributions Flatpak must be [pre-configured manually](https://flatpak.org/setup).*
+
+#### AppImage
 
 1. Go to [releases section](https://github.com/TGS963/upscayl/releases/latest)
 
 2. Download the `.AppImage` file.
 
 3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.
 
```

Comment:
```suggestion
Upscayl should be available on the Software Store of most Linux operating systems. You can download the Snap or [Flatpak](https://flatpak.org/setup) version, if your distro's Store App supports them.

#### Manual Install

1. Go to [releases section](https://github.com/upscayl/upscayl/releases/latest)

2. Download the `upscayl-x.x.x-linux.AppImage` file.

3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.

You can also choose to install using other formats like RPM (Fedora), DEB (Debian/Ubuntu based), and ZIP (Any Linux OS).
```

Reply:
I don't think an AppImage is a "Manual Install", which sounds like compiling it.

Reply:
> I don't think an AppImage is a "Manual Install", which sounds like compiling it.

Good point